server/tournament.py

```diff
@@ -22,6 +22,8 @@
 from server.utils import ordinal_suffix
 
 ROLES_ELIGIBLE_TO_SUBMIT_SCORES = ["captain", "admin", "spirit captain", "coach"]
+PLAYER_ROLE = "player"
+ASSISTANT_COACH_ROLE = "assistant coach"
```

Comment:
hmm where are we using this role? I saw that in UC that no teams are having this role hence excluded it

Reply:
i was using it for getting non playing teams, now that we're not showing, i'll remove this variable